java-extensions/hudi-reader/src/main/java/com/starrocks/hudi/reader/HudiColumnValue.java

```diff
@@ -159,4 +149,22 @@ public byte getByte() {
     public BigDecimal getDecimal() {
         return ((HiveDecimal) inspectObject()).bigDecimalValue();
     }
+
+    @Override
+    public LocalDate getDate() {
+        return LocalDate.parse(inspectObject().toString(), DATE_FORMATTER);
```

Comment:
There is still a format operation here; can it be removed?

what original type the object is？

Reply:
ALL DONE